### PR TITLE
[Spec] Change nntrainer install dir

### PR DESCRIPTION
**Pull request by zhoonit**
## Commits to be reviewed in this PR

<details><summary>[Spec] Change nntrainer install dir</summary><br />

This patch change nntrainer install dir to /usr/prefix/*,
see also https://github.com/nnstreamer/nnstreamer/issues/3560

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

</details>

### CODE REVIEW

File: nnstreamer/tensor_filter/meson.build [hunk 1]
```diff
@@ -24,7 +24,7 @@ shared_library('nnstreamer_filter_nntrainer',
   dependencies: nnstreamer_filter_nntrainer_deps,
   include_directories: [nntrainer_inc, '.'], # '.' shouldn't be installed
   install: true,
-  install_dir: filter_subplugin_install_dir
+  install_dir: '/usr/lib/nnstreamer'
```
Comment (myungjoo) [inline nnstreamer/tensor_filter/meson.build:27]: Keep this configurable (w/ meson_options?).
If a user wants to support multilib, there could be multiple instances of libnnstreamer_filter_nntrainer.so in a single device.

Comment (zhoonit) [reply nnstreamer/tensor_filter/meson.build]: Okay updated :)

**Review by lhs8928 — APPROVED** [review]
LGTM

File: packaging/nntrainer.spec [hunk 2]
```diff
@@ -353,7 +354,7 @@ meson --buildtype=plain --prefix=%{_prefix} --sysconfdir=%{_sysconfdir} \
       %{enable_gym} %{enable_nnstreamer_tensor_filter} %{enable_profile} \
       %{enable_nnstreamer_backbone} %{enable_tflite_backbone} \
       %{enable_tflite_interpreter} %{capi_ml_pkg_dep_resolution} \
-      %{enable_reduce_tolerance} %{enable_debug} build
+      %{enable_reduce_tolerance} -Dnnstreamer-subplugin-install-path=%{nnstreamer_subplugin_path} %{enable_debug}  build
```
Comment (kparichay) [inline packaging/nntrainer.spec:357]: minor : for readability, use %define

**Review by kparichay — APPROVED** [review]
Overall LGTM

File: packaging/nntrainer.spec (hunk 2 above)
Comment (zhoonit) [reply packaging/nntrainer.spec]: Done

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**